tests/NHSD.BuyingCatalogue.Identity.Api.Testing.Data/Entities/UserEntity.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+
+namespace NHSD.BuyingCatalogue.Identity.Api.Testing.Data.Entities
+{
+    public class UserEntity : EntityBase
+    {
+        public string PasswordHash { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string Email { get; set; }
+        public string PhoneNumber { get; set; }
+        public bool Disabled { get; set; }
+        public Guid OrganisationId { get; set; }
+        public string OrganisationFunction { get; set; }
+        public string Id { get; set; }
+
+        protected override string InsertSql => @"
+            INSERT INTO [dbo].[AspNetUsers]
+                       ([Id]
+                       ,[UserName]
+                       ,[NormalizedUserName]
+                       ,[Email]
+                       ,[NormalizedEmail]
+                       ,[EmailConfirmed]
+                       ,[PasswordHash]
+                       ,[SecurityStamp]
+                       ,[ConcurrencyStamp]
+                       ,[PhoneNumber]
+                       ,[PhoneNumberConfirmed]
+                       ,[TwoFactorEnabled]
+                       ,[LockoutEnabled]
+                       ,[AccessFailedCount]
+                       ,[PrimaryOrganisationId]
+                       ,[OrganisationFunction]
+                       ,[Disabled]
+                       ,[CatalogueAgreementSigned]
+                       ,[FirstName]
+                       ,[LastName])
+                 VALUES
+                       (@id
+                       ,@email
+                       ,UPPER(@email)
+                       ,@email
+                       ,UPPER(@email)
+                       ,1
+                       ,@passwordHash
+                       ,@passwordHash
+                       ,NEWID()
+                       ,@phoneNumber
+                       ,1
+                       ,0
+                       ,0
+                       ,0
+                       ,@organisationId
+                       ,@organisationFunction
+                       ,@disabled
+                       ,0
+                       ,@firstName
+                       ,@lastName)";
```

Comment:
SQL formatting: there's no need to delimit every identifier as it just hurts readability. Commas should be at the end of the line. I'm also not a fan of automatically placing every column on a new line – no doubt some will argue it should be done that way for the sake of readability. However, I think it can actually hurt readability and keeping columns grouped logically is more readable. For example:

```sql
INSERT INTO dbo.AspNetUsers
    (Id, UserName, NormalizedUserName, Email, NormalizedEmail, EmailConfirmed,
	PasswordHash, SecurityStamp, ConcurrencyStamp, PhoneNumber, PhoneNumberConfirmed,
	TwoFactorEnabled, LockoutEnabled, AccessFailedCount,
	PrimaryOrganisationId, OrganisationFunction, [Disabled], CatalogueAgreementSigned,
	FirstName, LastName)
VALUES
    (@id, @email, UPPER(@email), @email, UPPER(@email), 1,
	@passwordHash, @passwordHash, NEWID(), @phoneNumber, 1,
	0, 0, 0,
	@organisationId, @organisationFunction, @disabled, 0,
	@firstName, @lastName);
```
Every T-SQL statement should also be terminated with a semicolon – omitting the semicolon is deprecated.

Reply:
After seeing your example I'm inclined to agree on the grouping of parameters on lines, much easier to read